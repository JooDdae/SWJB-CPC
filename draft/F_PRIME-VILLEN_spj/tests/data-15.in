19311693679149518753502144214003786710913793392373971032687831373552165692609961375894759516755577392207517118732175328019712975417463958976173583985436367354797211721909176694567852116371987729713272237832933596199714159143975296983607943733433371917092718111983997916551684118117541143935945073947390793235303811673820914986412341739220746575119373311518081273114937207159758613467
