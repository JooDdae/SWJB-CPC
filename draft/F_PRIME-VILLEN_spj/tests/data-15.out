195
8 2 0 264 874 662 1092 673 601 552 993 29 0 2 22 2 544 4 2 25 27 390 23 0 2 4 2 3 3 1 0 849 379 1186 4 3 24 15 17 3 0 19 3 3 3 4 131 329 3 0 4 195 2 1 0 4 3 2 408 25 0 10 4 3 13 4 626 3 24 4 18 4 2 3 146 1066 830 2 3 92 4 1 0 0 4 1 0 1128 972 619 1062 38 4 300 4 10 4 0 2 1 4 1 1 2 4 23 10 2 2 17 806 4 0 13 17 0 85 4 3 428 23 793 14 4 2 2 14 2 2 4 8 0 127 1 4 42 0 8 146 1208 846 881 42 0 4 3 13 0 85 2 17 611 2 161 2 155 1154 703 141 19 2 1029 35 1076 1 2 13 131 329 630 3 0 8 2 4 2 2 0 0 3 0 1015 1 4 2 0 35 2 920 0 17 4 771 2 91 